accepted/PSR-7-http-message-meta.md

```diff
@@ -647,3 +647,12 @@ used to populate the headers of an HTTP message.
 * Anton Serdyuk
 * Phil Sturgeon
 * Chris Wilkinson
+
+## 7 Errata
+
+### 7.1 Invalid `UriInterface` after modification (2015/09/02)
+
+Supplying a valid value to a `UriInterface` modifying method does not guarantee that the resulting `UriInterface` 
+object will generate a valid URI string. Because no exception can be thrown in a `__toString` method, the invalid 
+URI string can only be detected when the new instance is created. In such situation, a `RuntimeException` exception 
```

Comment:
>> the issue is that you seem to think that an URI is a simple model

I mean the application model.

Very simple application with one component using URI and complex application with many components using it. In simple cases i prefer for example 'filter_var' and also add my own validation rules, etc. 

>> I don't understand the added value of a decorator 
>> What would a decorator adds ?

- Constitues and extracts custom validation layer,
- No duplication in complex model, when value object is not validated by itself, but components must relay on him.


Just I sad it's only my opinion about this errata :)


Regards


PS Whole validation after every "change"?

```php
use Zend\Diactoros\Uri;
$uri = new Uri('http://www.example.com');
$invalidUri = $uri->withHost('')->withPath('foo/bar');
```